src/Microsoft.Data.SqlClient/add-ons/Directory.Build.props

```diff
@@ -18,7 +18,7 @@
   <PropertyGroup>
     <TargetNetFxVersion Condition="'$(TargetNetFxVersion)' == ''">net462</TargetNetFxVersion>
     <TargetNetStandardVersion Condition="'$(TargetNetStandardVersion)' == ''">netstandard2.0</TargetNetStandardVersion>
-    <TargetNetCoreVersion Condition="'$(TargetNetCoreVersion)' == ''">netcoreapp3.1</TargetNetCoreVersion>
+    <TargetNetCoreVersion Condition="'$(TargetNetCoreVersion)' == ''">net6.0</TargetNetCoreVersion>
```

Comment:
Probably better to use the above suggestion or `Condition="$([MSBuild]::IsTargetFrameworkCompatible($(TargetFramework), 'net6.0'))"` (which will also match net7.0, net6.0-android etc)

Reply:
Not sure if this is going to achieve the goal we are expecting. Let's see what other team members and community contributors will say.

Reply:
My point was that as you upgrade versions, like moving to .net8, this expression continues to be true.

Reply:
> sounds good to me.

What part are you referring to?

Reply:
using `IsTargetFrameworkCompatible`